docs/reference/search/search-your-data/semantic-search-elser.asciidoc

```diff
@@ -97,27 +99,33 @@ PUT _ingest/pipeline/elser-v2-test
     {
       "inference": {
         "model_id": ".elser_model_2",
-        "target_field": "ml",
-        "field_map": { <1>
-          "text": "text_field"
-        },
-        "inference_config": {
-          "text_expansion": { <2>
-            "results_field": "tokens"
+        "input_output": [ <1>
+          {
+            "input_field": "content",
+            "output_field": "content_embedding"
           }
+        ],
+        "inference_config": {
+          "text_expansion": {} <2>
         }
       }
     }
   ]
 }
 ----
-// TEST[skip:TBD]
-<1> The `field_map` object maps the input document field name (which is `text`
-in this example) to the name of the field that the model expects (which is
-always `text_field`).
-<2> The `text_expansion` inference type needs to be used in the {infer} ingest
+<1> Configuration object that defines the `input_field` for the {infer} process 
+and the `output_field` that will contain the {infer} results.
+<2> The `text_expansion` {infer} type needs to be used in the {infer} ingest
```

Comment:
This is no longer true. It was required in the past to set the `results` field to `tokens` but not the output field is specified as `content_embedding`

The full config is now
```
PUT _ingest/pipeline/elser-v2-test
{
  "processors": [
    {
      "inference": {
        "model_id": ".elser_model_2_linux-x86_64",
        "input_output": [ 
          {
            "input_field": "content",
            "output_field": "content_embedding"
          }
        ]
      }
    }
  ]
}
```